### PR TITLE
Update rollback docs

### PR DESCRIPTION
**Pull request by fredericfran-gds**
Changes:
1. delete reference about rollbacks being applicable to `integration`
   only
2. add a section about re-enabling image updates

### CODE REVIEW

File: source/manage-app/roll-back-app/index.html.md.erb [hunk 1]
```diff
@@ -31,7 +31,9 @@ Once you have merged your pull request, [Argo CD automatically deploys the older
 
 ## Triggering the deploy GitHub Action to deploy an older release
 
-This rollback method only works for the integration environment.
+**Warning**
```
Comment (jonathanglassman) [inline source/manage-app/roll-back-app/index.html.md.erb:34]: GOV.UK style is to only use bold format for UI text e.g. "Select __Submit__ to submit the report." Would recommend not using the word warning at all.

File: source/manage-app/roll-back-app/index.html.md.erb [hunk 2]
```diff
@@ -31,7 +31,9 @@ Once you have merged your pull request, [Argo CD automatically deploys the older
...
+Any manual deploy of a given app will freeze the automatic update of its images,
+this can be enabled by following the procedure in the next section.
```
Comment (jonathanglassman) [inline source/manage-app/roll-back-app/index.html.md.erb:36]: Would recommend re-writing to make active, like the rest of the content. I.e.

"When you manually trigger the deploy GitHub Action for an app, that will stop the automatic update of that app's images. You can re-enable the automatic image update by following the [documentation on re-enabling automatic image updates after manual deploys](#re-enabling-automatic-image-updates-after-manual-deploys).

File: source/manage-app/roll-back-app/index.html.md.erb [hunk 3]
```diff
@@ -49,3 +51,11 @@ Argo CD will then:
 - deploy that version of the app
 
 Check the [#govuk-deploy-alerts Slack channel](https://gds.slack.com/archives/C01EE7US9R6) for a notification when the older version of the app is successfully deployed.
+
+## Re-enabling automatic image updates after manual deploys
```
Comment (jonathanglassman) [inline source/manage-app/roll-back-app/index.html.md.erb:55]: New section LGTM

**Review by jonathanglassman — CHANGES_REQUESTED** [review]
Content looks good overall. Have suggested a couple of changes.

File: source/manage-app/roll-back-app/index.html.md.erb (hunk 1 above)
Comment (fredericfran-gds) [reply source/manage-app/roll-back-app/index.html.md.erb:34]: ok will do

File: source/manage-app/roll-back-app/index.html.md.erb (hunk 2 above)
Comment (fredericfran-gds) [reply source/manage-app/roll-back-app/index.html.md.erb:36]: will do, thanks

File: source/manage-app/roll-back-app/index.html.md.erb (hunk 3 above)
Comment (fredericfran-gds) [reply source/manage-app/roll-back-app/index.html.md.erb:55]: thanks

**Comment by fredericfran-gds** [comment]
@jonathanglassman if you could have a look, I made the changes you recommended. Thanks!

**Review by jonathanglassman — APPROVED** [review]

**Pull request merged**